### DISCUSSION TITLE
Dependabot ignores Containerfile

### DISCUSSION THREAD

**Discussion by Wivik** [community/community#54288] [Code Security] [answered]
### Select Topic Area

Product Feedback

### Body

Hello,

As a Podman user, I'm used to their naming conventions such as Containerfile instead of Dockerfile.

However, when using the Version check of Dependabot, the tool search only for a Dockerfile. And I'm not sure if the Security update has the same behavior.

![image](https://user-images.githubusercontent.com/48727868/235774984-1ff275a0-d90b-4914-abc9-fd73808a948b.png)

It would be great to support other names for the container build instructions files.

Thanks

**Comment by carogalvin** [comment, marked as answer]
Hi @Wivik, thanks for that feedback! We have an open issue for this in the `dependabot-core` repo. This is not currently on our roadmap, but you can leave a comment or watch that issue to get updates: https://github.com/dependabot/dependabot-core/issues/6067

**Comment by Wivik** [comment]
Hello @carogalvin 

I've forgot to check on Dependabot's repository, shame on me ! 

Thanks for the issue sharing :)
